source/read/project.txt

```diff
@@ -0,0 +1,126 @@
+.. _kotlin-sync-project:
+
+========================
+Specify Fields To Return
+========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, filter, project, select
+
+Overview
+--------
+
+In this guide, you can learn how to specify which fields to return from a read
+operation by using a **projection**. A projection is a document that specifies
+which fields MongoDB returns from a query.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+The documents in this collection are modeled by the following {+language+} data class:
+
+.. literalinclude:: /includes/read/project.kt
+   :start-after: start-data-class
+   :end-before: end-data-class
+   :language: kotlin
+   :copyable:
+
+Projection Types
+----------------
+
+You can use a projection to specify which fields to include in a return
+document, or to specify which fields to exclude. You cannot combine inclusion and
```

Comment:
Q: I'm unclear on whether "You cannot combine inclusion and exclusion statements..." means that I can combine both only if I am excluding the ``_id`` field and nothing else (in my exclusion statement), or if I can combine both if I have an exclusion statement that excludes the ``_id`` field as well as excluding other fields.

Reply:
C: Ok, reading further I now understand that the exclusion statement only applies to _id. You could potentially make that clearer earlier in the document.

Reply:
C: This also confuses me because it seems like, based on my understanding, the only time you'd combine includes and excludes statements is to exclude the _id field, as all other fields are implicitly excluded when you use a includes statement? Could you clarify when someone would want to use both an includes and excludes that didn't just exclude _id?

Reply:
The idea is that a standard query will pull all the document's fields, and once you apply a projection to specify the inclusion of a single field, all other fields automatically get excluded (except "_id").

Someone _shouldn't_ want to use both an includes and excludes. I can try addressing this confusion by pulling up the clarifying text from below the page into this mention up here.

Reply:
yeah, I think moving the clarifying text up would be good.